src/App.css

```diff
@@ -1,39 +1,116 @@
-.App {
-  text-align: center;
+.nav-wrap {
+  background-color: green;
+  margin: 0;
+  display: flex;
+  justify-content: space-between;
+}
+
+li {
+  list-style: none;
+  margin: 0;
+}
+
+.nav-wrap p {
+  color: white;
+  font-size: 30px;
+  font-weight: 600;
+  margin-left: 15px;
+}
+
+.nav-wrap ul {
+  display: flex;
+  justify-content: flex-end;
+  margin-right: 40px;
```

Comment:
- [ ] [OPTIONAL] Styling is not a requirement at this stage, Microverse will give you a design to follow when it's time for styling.

Reply:
ok